### PR TITLE
Add Abs 13 operator

### PR DESCRIPTION
**Pull request by goodsw4all**
This pull request is part of the work to make operators for connx.

- Abs 13 as a practice

### CODE REVIEW

File: src/opset/Abs_13.c [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                INT8, INT16, INT32, INT64,
+                FLOAT16, FLOAT32, FLOAT64
+            ]
+         %}*/
+        /*{% for DTYPE, TYPE in loop_types(*type_constraints) %}*/
+        /*{%    if "FLOAT" in DTYPE %}*/
+        /*{%        set abs_func = 'fabs' %}*/
+        /*{%    elif "UINT" in DTYPE %}*/
+        /*{%        set abs_func = '' %}*/
+        /*{%    elif "INT64" in DTYPE %}*/
```
Comment (tribela) [inline src/opset/Abs_13.c:49]: For UINT64|INT64?

```suggestion
        /*{%    elif DTYPE in (UINT64, INT64) %}*/
```

But UINT64 is selected by L47, So it should be
```suggestion
        /*{%    elif DTYPE == INT64 %}*/
```

File: src/opset/Abs_13.c [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        /*{% set type_constraints = [
+                UINT8, UINT16, UINT32, UINT64,
+                INT8, INT16, INT32, INT64,
+                FLOAT16, FLOAT32, FLOAT64
+            ]
+         %}*/
+        /*{% for DTYPE, TYPE in loop_types(*type_constraints) %}*/
+        /*{%    if "FLOAT" in DTYPE %}*/
+        /*{%        set abs_func = 'fabs' %}*/
+        /*{%    elif "UINT" in DTYPE %}*/
```
Comment (tribela) [inline src/opset/Abs_13.c:47]:
```suggestion
        /*{%    elif DTYPE in (UINT8, UINT16, UINT32, UINT64) %}*/
```
 or
```suggestion
        /*{%    elif DTYPE.startswith('UINT') %}*/
```

File: src/opset/Abs_13.c [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+    switch (X->dtype) {
+        /*{% set type_constraints = [
+                UINT8, UINT16, UINT32, UINT64,
+                INT8, INT16, INT32, INT64,
+                FLOAT16, FLOAT32, FLOAT64
+            ]
+         %}*/
+        /*{% for DTYPE, TYPE in loop_types(*type_constraints) %}*/
+        /*{%    if "FLOAT" in DTYPE %}*/
```
[the remark this answers is not in the document]
Comment (goodsw4all) [reply src/opset/Abs_13.c:45]: Thanks, I try some change as your advice.
Can you please explain to use memcpy instead assignment?
Is it more optimized way?

[the remark this answers is not in the document]
Comment (goodsw4all) [reply src/opset/Abs_13.c:45]: It became more readable after moving if-clauses ✨

[the remark this answers is not in the document]
Comment (tribela) [reply src/opset/Abs_13.c:45]: > Thanks, I try some change as your advice. Can you please explain to use memcpy instead assignment? Is it more optimized way?

Yes. using if data are exactly same, memcpy is way faster than for-loop (See concat operator)

**Pull request merged**